resources/views/components/order-preview.blade.php

```diff
@@ -1,7 +1,7 @@
 <div class="w-full sm:flex sm:max-w-none border border-gray-300 dark:border-gray-700 rounded-xl overflow-hidden">
     <div
         class="h-48 sm:h-auto sm:w-48 flex-none bg-cover text-center overflow-hidden"
-        style="background-image: url({{ $order->orderLines->first()->product_image_path }})" title="{{--TODO!--}}">
+        style="background-image: url({{ asset($order->orderLines->first()->product_image_path) }})" title="{{--TODO!--}}">
```

Comment:
Zet in deze div een `<img src="{{ asset_url($order->orderLines->first()->product_image_path) }}">`
En zet er een alt tekst bij (kijk hoe dat bij product.blade.php gedaan is)